ui/src/app/views/workspace-list/component.tsx

```diff
@@ -246,6 +246,12 @@ export const WorkspaceList = withUserProfile()
     this.reloadWorkspaces();
   }
 
+  componentDidUpdate() {
```

Comment:
this fixes a little bug I noticed when testing deletion that the create workspace button on the list workspaces page never updated to be active when billing proj loaded